app/src/main/java/org/jellyfin/androidtv/integration/dream/LibraryDreamService.kt

```diff
@@ -26,52 +57,203 @@ import org.jellyfin.sdk.model.api.ImageFormat
 import org.jellyfin.sdk.model.api.ImageType
 import org.jellyfin.sdk.model.constant.ItemSortBy
 import org.koin.android.ext.android.inject
+import org.koin.androidx.compose.get
 import timber.log.Timber
-import java.util.concurrent.ExecutionException
+import kotlin.time.Duration.Companion.seconds
+
+fun Modifier.overscan(): Modifier = then(padding(48.dp, 27.dp))
+
+@Composable
+fun DreamHeader(
+	showLogo: Boolean,
+	showClock: Boolean,
+) {
+	Row(
+		horizontalArrangement = Arrangement.SpaceBetween,
+		modifier = Modifier
+			.fillMaxWidth()
+			.overscan(),
+	) {
+		// Logo
+		AnimatedVisibility(
+			visible = showLogo,
+			enter = fadeIn(),
+			exit = fadeOut(),
+			modifier = Modifier.height(41.dp),
+		) {
+			Image(
+				painter = painterResource(R.drawable.app_logo),
+				contentDescription = stringResource(R.string.app_name),
+			)
+		}
+
+		Spacer(
+			modifier = Modifier
+				.fillMaxWidth(0f)
+		)
+
+		// Clock
+		AnimatedVisibility(
+			visible = showClock,
+			enter = fadeIn(),
+			exit = fadeOut(),
+		) {
+			Text(
+				text = DateUtils.formatDateTime(LocalContext.current, System.currentTimeMillis(), DateUtils.FORMAT_SHOW_TIME),
```

Comment:
The clock doesn't update right now, so it stays stuck on the time the daydream started. Fix that and make it a separate composable so it's reusable.